csharp/sources/Valkey.Glide/Valkey.Glide.csproj

```diff
@@ -1,76 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="$(MSBuildThisFileDirectory)../../props/version.props"/>
+  <Import Project="$(MSBuildThisFileDirectory)../../props/common.props"/>
+
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
     <RootNamespace>Valkey.Glide</RootNamespace>
     <AssemblyName>Valkey.Glide</AssemblyName>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-    <LangVersion>12.0</LangVersion>
-    <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
-    <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
-    <EnableNETAnalyzers>false</EnableNETAnalyzers>
-    <NuGetAudit>false</NuGetAudit>
-    <WarningLevel>0</WarningLevel>
-    <Configurations>Debug;Release;Lint</Configurations>
-  </PropertyGroup>
-
-  <!-- Enable analysis and un-suppress warnings only in `Lint` configuration -->
-  <PropertyGroup Condition="'$(Configuration)' == 'Lint'">
-    <LangVersion>12.0</LangVersion>
-    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
-    <RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
-    <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    <NuGetAudit>true</NuGetAudit>
-    <WarningLevel>4</WarningLevel>
-    <AnalysisLevel>latest</AnalysisLevel>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-  </PropertyGroup>
-
-  <!-- Workaround for https://github.com/dotnet/roslyn/issues/41640 -->
-  <PropertyGroup>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <NoWarn>$(NoWarn);CS1591;CS1573;CS1587</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
-    <InternalsVisibleTo Include="Valkey.Glide.IntegrationTests" />
-    <InternalsVisibleTo Include="Valkey.Glide.UnitTests" />
-  </ItemGroup>
-
-  <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
-    <Exec Condition="'$(Configuration)' == 'Debug'"
-          Command="cargo build"
-          EnvironmentVariables="CARGO_TERM_COLOR=always"
-          ConsoleToMSBuild="true"
-          WorkingDirectory="$(MSBuildThisFileDirectory)../../rust">
-      <Output TaskParameter="ConsoleOutput" PropertyName="OutputOfExec" />
-    </Exec>
-    <Exec Condition="'$(Configuration)' == 'Release'"
-          Command="cargo build --release"
-          EnvironmentVariables="CARGO_TERM_COLOR=always"
-          ConsoleToMSBuild="true"
-          WorkingDirectory="$(MSBuildThisFileDirectory)../../rust">
-      <Output TaskParameter="ConsoleOutput" PropertyName="OutputOfExec" />
-    </Exec>
-  </Target>
+    <!-- ToDo: Clean up for proper packing
```

Comment:
Do you plan to complete this TODO in this PR or in a next one?

Reply:
See full comment. Really we have to decide on what assets to include by default for development and for deployment (aka: nuget).
Cannot decide that alone imo

Reply:
We need to add nuspec file and other candidates:
- `version.json`
- `Versions.props`
- `Directory.Build.targets`
- `Directory.Build.props`

Reply:
Probably should postpone for a later PR